packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -495,4 +514,19 @@ export enum LaunchType {
    * The service will be launched using the FARGATE launch type
    */
   FARGATE = 'FARGATE'
+}
+
+/**
+ * Propagate tags from either service or task definition
+ */
+export enum PropagateTagsFromType {
```

Comment:
Maybe we could name this `PropagatedTagSource`?

Reply:
Good point. I agree.